python_modules/dagster/dagster/_config/pythonic_config/config.py

```diff
@@ -163,13 +204,14 @@ def __init__(self, **config_dict) -> None:
         modified_data = {}
         for key, value in config_dict.items():
             field = self.__fields__.get(key)
-            if field and not field.required and value is None:
+            field = model_fields(self).get(key)
+            if field and not field.is_required() and value is None:
```

Comment:
hmm didn't you just alter this downstack, what's going on here?

Reply:
think this is resolved by a rebase